src/gpu/intel/ocl/reusable_softmax.cl

```diff
@@ -72,3 +73,159 @@ reusable_softmax_fwd_generic(__global SRC_DATA_T *src, __global DST_DATA_T *dst,
         dst[c - begin] = TO_DST(unscaled * scale);
     }
 }
+#endif
+
+#define VECT_SIZE 8
+#define DIVUP(a, b) (((a) + (b)-1) / (b))
```

Comment:
You can reuse the implementation at: https://github.com/uxlfoundation/oneDNN/blob/2f841141380d63f89e953da8ead06e94cc8c9bd9/src/gpu/intel/ocl/ocl_math_utils.h#L30-L36

Reply:
Yeah that's the idea - - dealing with a CI issue at the moment.